tests/abort-or-stop.cpp

```diff
@@ -177,7 +177,6 @@ main()
         EXPECT(packet.result_ == true, "Something went wrong in 'abort' test.");
 
         // stop waits until finished
-        CHECK(runtime = acquire_init(reporter));
```

Comment:
Do you have an idea of why this segfaulted? 

We should have some defined behavior on double init (controlled failure).

Reply:
No, but I couldn't reproduce it locally.

Reply:
good point